keyboards/illusion/rosa/info.json

```diff
@@ -5,7 +5,7 @@
     "width": 15, 
     "height": 5, 
     "layouts": {
-        "LAYOUT_60_ansi": {
+        "LAYOUT_60_ansi_tsangan": {
```

Comment:
should also add this to `LAYOUTS`, I think?

Reply:
Done.